python/builtInExample2.py

```diff
@@ -0,0 +1,144 @@
+
+# create a new 'Extract Edges'
+extractEdges1 = ExtractEdges(Input=tTKProjectionFromField1)
+
+# create a new 'Tube'
+tube1 = Tube(Input=extractEdges1)
+tube1.Scalars = ['POINTS', 'log(Rho)']
+tube1.Vectors = ['POINTS', 'Normals']
+tube1.Radius = 0.02
+
+# create a new 'TTK ProjectionFromField' to project the 'log(s)' contour onto the range space
+tTKProjectionFromField2 = TTKProjectionFromField(Input=contour3)
+tTKProjectionFromField2.UComponent = ['POINTS', 'log(Rho)']
+tTKProjectionFromField2.VComponent = ['POINTS', 'log(s)']
+
+# create a new 'Extract Edges'
+extractEdges2 = ExtractEdges(Input=tTKProjectionFromField2)
+
+# create a new 'Tube'
+tube2 = Tube(Input=extractEdges2)
+tube2.Scalars = ['POINTS', 'log(Rho)']
+tube2.Vectors = ['POINTS', 'Normals']
+tube2.Radius = 0.02
+
+# create a new 'Tetrahedralize'
+tetrahedralize1 = Tetrahedralize(Input=example2vti)
+
+# Four manually created polylines in the range space to select regions corresponding to 
+# Oxygen and carbon atoms, followed by covalent and non-covalent bonds
+rangePolygonsCoordinates = [
+    [1.89657, -0.278516, 0.0, 2.06003, -0.24493, 0.0, 2.15538, -0.278516, 0.0, 2.21441, -0.416216, 0.0],
+    [1.3744, 0.000242441, 0.0, 1.51061, -0.00983316, 0.0, 1.65137, -0.0434185, 0.0, 1.70586, -0.181118, 0.0],
+    [1.21547, -0.701691, 0.0, 1.27904, -0.614369, 0.0, 1.51061, -0.607652, 0.0, 1.52878, -1.16853, 0.0, 1.53786, -0.688257, 0.0],
+    [-0.310174, -0.325535, 0.0, -0.0150337, -0.124023, 0.0, 0.216538, -0.140816, 0.0, 0.466272, -0.399423, 0.0]
+]
+
+# Save the four range space control polygons
+polygonTubes = []
+
+# Save the corresponding four fiber surfaces
+fiberSurfaces = []
+
+for coords in rangePolygonsCoordinates:
+    # create a new 'Poly Line Source'
+    polyLineSource1 = PolyLineSource()
+    polyLineSource1.Points = coords
+    
+    # create a new 'Resample With Dataset'
+    resampleWithDataset1 = ResampleWithDataset(SourceDataArrays=tTKContinuousScatterPlot1,
+        DestinationMesh=polyLineSource1)
+    resampleWithDataset1.CellLocator = 'Static Cell Locator'
+
+    # create a new 'Tetrahedralize'
+    tetrahedralize2 = Tetrahedralize(Input=resampleWithDataset1)
+
+    # create a new 'TTK FiberSurface'
+    tTKFiberSurface1 = TTKFiberSurface(InputDomain=tetrahedralize1,
+        RangePolygon=tetrahedralize2)
+    tTKFiberSurface1.DomainUComponent = ['POINTS', 'log(Rho)']
+    tTKFiberSurface1.DomainVComponent = ['POINTS', 'log(s)']
+    tTKFiberSurface1.PolygonUComponent = ['POINTS', 'log(Rho)']
+    tTKFiberSurface1.PolygonVComponent = ['POINTS', 'log(s)']
+    tTKFiberSurface1.WithPointMerging = 1
+
+    # create a new 'Generate Surface Normals'
+    generateSurfaceNormals1 = GenerateSurfaceNormals(Input=tTKFiberSurface1)
+    
+    fiberSurfaces.append(generateSurfaceNormals1)
+
+    # create a new 'Extract Surface'
+    extractSurface1 = ExtractSurface(Input=resampleWithDataset1)
+
+    # create a new 'Tube'
+    tube3 = Tube(Input=extractSurface1)
```

Comment:
I would suggest to store the raw geometry instead of graphical primitives (tubes).
(no need to create the tubes here)